src/DocumentDb/Microsoft.Omex.DocumentDb.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetStandardVersion)</TargetFrameworks>
+    <Description>Contains Omex Document Db code</Description>
+    <RootNamespace>Microsoft.Omex.DocumentDb</RootNamespace>
+    <AssemblyName>Microsoft.Omex.DocumentDb</AssemblyName>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include="Readme.txt" PackagePath="%(Identity)" />
```

Comment:
I think a readme.md would be better.
Also, I'd suggest fleshing out the readme s bit as part of this pull request.